Example/TrustlySDK/Base.lproj/Main.storyboard

```diff
@@ -45,19 +45,18 @@
                                 </constraints>
                             </view>
                             <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="0Sh-Ok-y02">
-                                <rect key="frame" x="34" y="729" width="324" height="35"/>
+                                <rect key="frame" x="34" y="717" width="324" height="35"/>
                                 <color key="backgroundColor" red="0.38340717549999997" green="0.80894964930000002" blue="0.0064765461719999999" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
                                 <constraints>
-                                    <constraint firstAttribute="height" constant="35" id="V4B-gC-WZy"/>
-                                    <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="324" id="a72-UG-B9y"/>
+                                    <constraint firstAttribute="height" constant="35" id="sWW-eC-Rku"/>
                                 </constraints>
                                 <fontDescription key="fontDescription" type="boldSystem" pointSize="17"/>
                                 <inset key="imageEdgeInsets" minX="0.0" minY="0.0" maxX="2.2250738585072014e-308" maxY="0.0"/>
-                                <state key="normal" title="Pay">
+                                <state key="normal" title="Pay with AESWebAuth">
                                     <color key="titleColor" systemColor="secondarySystemBackgroundColor"/>
                                 </state>
                                 <connections>
-                                    <action selector="pay:" destination="vXZ-lx-hvc" eventType="touchUpInside" id="eGD-FN-iNW"/>
+                                    <action selector="payWituAESWebAuth:" destination="vXZ-lx-hvc" eventType="touchUpInside" id="LyG-e4-6SZ"/>
```

Comment:
Maybe there's a typo here in the word `with`?

Reply:
Fixed